pkg/apis/admissionregistration/types.go

```diff
@@ -566,6 +566,19 @@ type ValidatingWebhook struct {
 	// does not understand, calls to the webhook will fail and be subject to the failure policy.
 	// +optional
 	AdmissionReviewVersions []string
+
+	// MatchConditions is a list of conditions on the AdmissionRequest ('request') that must be met
+	// for a request to be sent to this webhook. All conditions in the list must evaluate to TRUE for
+	// the request to be matched. This is an alpha feature and managed by the AdmissionWebhookMatchConditions feature gate.
+	// If this field is not set no matching will occur through cel. If any condition in the list fails to be met apiserver
+	// will not call out to the webhook.
+	//
+	// Defaults to nil
+	// +optional
+	// +patchMergeKey=name
+	// +patchStrategy=merge
+	// +listType=atomic
```

Comment:
I think you don't need these on the internal types

Reply:
removed